packages/plugin-compiler/src/main/kotlin/io/realm/kotlin/compiler/AccessorModifierIrGeneration.kt

```diff
@@ -238,9 +254,13 @@ class AccessorModifierIrGeneration(private val pluginContext: IrPluginContext) {
                             collectionType = CollectionType.NONE
                         )
                         modifyAccessor(
-                            declaration,
-                            getFunction = getValue,
-                            setFunction = setValue
+                            property = declaration,
+                            getFunction = getByteArray,
+                            fromRealmValue = null,
+                            toPublic = null,
```

Comment:
`toPublic/fromPublic` are always null, as they are no longer used we could remove it.